apps/health_campaign_field_worker_app/lib/utils/utils.dart

```diff
@@ -80,18 +80,18 @@ class CustomValidator {
         : {'required': true};
   }
 
-  static Map<String, dynamic>? validMobileNumber(
```

Comment:
Hi Efe,
I think we prefer this version with variable name control instead of the other one.
Can you please use this one that uses better variable names?

The same apply for startsWith7or9 function bellow.